documentation/graphing-overview/index.html.md

```diff
@@ -0,0 +1,279 @@
+---
+title: "Flow.js and Data Visualization: Graphing with Contour"
+layout: "flow"
+isPage: true
+---
+
+##Flow.js and Data Visualization: Graphing with Contour
+
+
+Flow.js makes it easy to show data from your model variables in [text](../generated/dom/attributes/binds/default-bind-attr/) or [lists](../generated/dom/attributes/foreach/default-foreach-attr/). However, often you want more than just textual output &mdash; you want to visualize the results of your simulation!
```

Comment:
either is correct and "in" seems more natural to me ... i checked google for "show data in" (335k) vs "show data through" (43k) to confirm.